src/python/grpcio_tests/tests/qps/README.md

```diff
@@ -0,0 +1,100 @@
+# Python Benchmark Tools
+
+## Scenarios
+
+In `src/proto/grpc/testing/control.proto`, it defines the fields of a scenario.
+In `tools/run_tests/performance/scenario_config.py`, the script generates actual scenario content that usually in json format, or piped to another script.
+
+All Python related benchmark scenarios are:
+* netperf
+* python_generic_sync_streaming_ping_pong
+* python_protobuf_sync_streaming_ping_pong
+* python_protobuf_async_unary_ping_pong
+* python_protobuf_sync_unary_ping_pong
+* python_protobuf_sync_unary_qps_unconstrained
+* python_protobuf_sync_streaming_qps_unconstrained
+* python_protobuf_sync_unary_ping_pong_1MB
+
+Here I picked the top 2 most representative scenarios of them, and reduce their benchmark duration from 30 seconds to 10 seconds:
+* python_protobuf_async_unary_ping_pong
+* python_protobuf_sync_streaming_ping_pong
+
+## Why keep the scenario file if it can be generated?
+
+Well... The `tools/run_tests/performance/scenario_config.py` is 1274 lines long. The intention of building these benchmark tools is reducing the complexity of existing infrastructure code. So, instead of calling layers of abstraction to generate the scenario file, keeping a valid static copy is more preferable.
```

Comment:
s/more preferable/preferable/

Optional: Add a "future-proofing" sentence like - "If the use case for this tool grows beyond these two scenarios, we can incorporate automatic generation and selection of scenarios into the tool."

Reply:
Added:

Also, if the use case for this tool grows beyond simple static scenarios, we can incorporate automatic generation and selection of scenarios into the tool.